### PR TITLE
chore: security updates 23.03.22

### PR DESCRIPTION
**Pull request by ingridklepsvik**
### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

-   [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
-   [x] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] I have updated the documentation accordingly
-   [ ] I have added tests to cover my changes

### Description

I have updated the dependencies except for react-sortablejs.
React-sortablejs upgrade  - import error on build. Needs further investigation.

image

Before update:
Pasted Graphic

After update: 
image

### CODE REVIEW

**Comment by CsabaTheKiss** [comment]
Checked in storybook, looks good!

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "license": "MIT",
     "peerDependencies": {
         "@equinor/echo-utils": "^0.2.7",
-        "@equinor/eds-core-react": "^0.17.0",
+        "@equinor/eds-core-react": "^0.18.0",
```
Comment (CsabaTheKiss) [inline package.json:32]: Probably a leftover: this should be reverted back to 0.17.0 - as discussed, we upgrade EDS in a separate task.

**Comment by CsabaTheKiss** [comment]
For me, running lint throws an error: 🤔 
Do you get the same result?
image

File: package.json (hunk 1 above)
Comment (ingridklepsvik) [reply package.json]: Downgraded it. It was a leftover

**Comment by ingridklepsvik** [comment]
> For me, running lint throws an error: 🤔 Do you get the same result? image

Yes, I got the same error. I did some changes to the config file now, so it works again.

I see that all of the configs was merged into one in version 8.0.0. So we just need the "prettier" config now, witch includes ESLint rules and rules from all plugins.

Changelog for v.8: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

**Review by CsabaTheKiss — APPROVED** [review]
Checked via storybook, looks good to me! 💯

**Pull request merged**